Removes a trailing slash at the end of a URI which is required not
to have a trailing slash, which was discovered via a
NullPointerException in the redfish-generator.
--- a/DSP8010_2024.2/openapi/openapi.yaml
+++ b/DSP8010_2024.2/openapi/openapi.yaml
@@ -141422,7 +141422,7 @@
       required: true
       schema:
         type: string
-  /redfish/v1/ThermalEquipment/HeatExchangers/{CoolingUnitId}/LeakDetection/LeakDetectors/:
+  /redfish/v1/ThermalEquipment/HeatExchangers/{CoolingUnitId}/LeakDetection/LeakDetectors:
     get:
       responses:
         '200':
